### ISSUE TITLE
`web_install_scylla` doesn't work on master

### ISSUE THREAD

**Issue by fruch** [scylladb/scylla-cluster-tests#4389]
Seem like it's assuming to work only on release branches, and doesn't take master/enterprise branches into account
```
        if match := re.match(r'\D*(\d+\.\d+)', scylla_version or git_branch):
            version = f"nightly-{match.group(1)}"
        else:
            raise Exception("Scylla version for web install isn't identified")
```

**Comment by yarongilor** [comment]
fixed by https://github.com/scylladb/scylla-cluster-tests/pull/4398

**Comment by fgelcer** [comment]
> fixed by #4398

@yarongilor , when you add to your commit message `fixes #<issue number>` it closes automatically the issue, so no action needed for it...
